### PR TITLE
raftstore-v2: add some logs and fix possible race between clean and tick

### PR DESCRIPTION
**Pull request by tabokie**
### What is changed and how it works?

Issue Number: Ref #14386

What's Changed:

```commit-message
None
```

### Check List 

Tests 

- Unit test

### Release note 

```release-note
None
```

### CODE REVIEW

File: components/raftstore-v2/src/operation/ready/mod.rs [hunk 1]
```diff
@@ -934,6 +934,25 @@ impl<EK: KvEngine, ER: RaftEngine> Storage<EK, ER> {
         let prev_raft_state = self.entry_storage().raft_state().clone();
         let ever_persisted = self.ever_persisted();
 
+        // If snapshot initializes the peer, we don't need to write apply trace again.
+        if !ever_persisted {
```
Comment (BusyJay) [inline components/raftstore-v2/src/operation/ready/mod.rs:938]: The comment above line explains `self.ever_persisted()` is used.

Comment (tabokie) [reply components/raftstore-v2/src/operation/ready/mod.rs:938]: Oh, so you mean this block should be placed after `apply_snapshot`. It's really not that obvious especially when `ever_persisted` is cached and used later in the function.

**Review by BusyJay — APPROVED** [review]

**Review by tonyxuqqi — APPROVED** [review]

**Pull request merged**